addons/dark-www/addon.json

```diff
@@ -1863,6 +1863,114 @@
         },
         "a": 0.74
       }
+    },
+    {
+      "name": "forum-bold-button",
+      "value": {
+        "type": "textColor",
+        "black": "url(https://cdn.scratch.mit.edu/scratchr2/static/__e65d8a7c03267938d906e22792610478__//djangobb_forum/js/markitup/sets/bbcode/images/bold.png)",
+        "white": "url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAhFBMVEUAAADe3t7T09POzs7s7Ozg4ODn5+fh4eHY2Njz8/PT09P19fX7+/vv7+/c3Nz4+PjV1dXOzs7g4ODa2trLy8v39/fu7u7s7Ozo6OjT09Po6Ojm5ub6+vr4+Pjl5eXY2Njl5eXLy8vu7u7h4eHj4+Pd3d3a2trX19fS0tLz8/Pm5ubMzMxZSeJAAAAAInRSTlMAf79/z8+/v5+Pf3AgEBDv7+/f38+/v7+vr5+fj39/f2BgO3gytQAAAGJJREFUGNOdykUOgDAARNEp7u5Ocbj//SAEGtj27+ZlwFcfHb6Ye5rDRFwFQJvlP5BpZGAsApxgKD8QEnlQwGpnFSh2yX6BTBfYlJoM7vtG6xeaUQJM6lrP7rI00fW4ssDXCeLqBY1pf6pbAAAAAElFTkSuQmCC)",
```

Comment:
Cool solution, but the addon.json is already really big. Could separate PNG files be used instead?

Reply:
I have already tried that, but neither %addon-self-dir%/assets/file.png nor ./assets/file.png worked, so I went with this instead.

Reply:
The value of a `customCssVariable` can reference other CSS variables. So you can set something like this in the CSS file:
```css
:root {
  --darkWww-forumBoldButtonBlack: url("https://cdn.scratch.mit.edu/scratchr2/static/djangobb_forum/js/markitup/sets/bbcode/images/bold.png");
  --darkWww-forumBoldButtonWhite: url("%addon-self-dir%/assets/forum_buttons/bold_white.png");
}
```

And this in the `addon.json`:
```json
"black": "var(--darkWww-forumBoldButtonBlack)",
"white": "var(--darkWww-forumBoldButtonWhite)",
```

Reply:
That's a lot better than my solution, I'll do that.

Reply:
Done, but I'm not sure how I'd test it now because I don't know how to download a branch.